Patch is.Sigma

tfjaeger opened the pull request:
Added ability to handle list inputs. Shoutout to Orhun Ulusahin for identifying this issue.

ShawnNCummings commented on R/class.R:
stop2() isn't defined as far as I can tell in this or any required packages-- could either be added (it's only 3 lines), or one could load JASPgraphs

tfjaeger replied:
It's a misc function that I've copied from brms in another branch. Great catch. Thank you. I've added it now.

tfjaeger commented:
> I don't have a lot of confidence in my ability to assess the changes made to .stan files, and see comments on slack.

No need to check the stan updates. Those are changes from another branch that I needed to port over since Stan now requires them (they were optional for a long time).

Do you mind testing once more whether the addition of the noise_treatment and lapse_treatment arguments addresses the remaining issues you had identified?
